documentation/static/tutorials/flux-getting-started.md

```diff
@@ -0,0 +1,142 @@
+Nightly builds for each are available on the [InfluxData Downloads](https://portal.influxdata.com/downloads) page.
+Once downloaded and unpackaged, move the binaries into your system's `$PATH`.
+
+### Configure InfluxDB to use Flux
+Flux is enabled by default in your `influxdb.conf`, the location of which
+[depends on your operating system](https://docs.influxdata.com/influxdb/v1.5/administration/config/#using-the-configuration-file).
+
+In the default config generated by the nightly builds, Flux is configured under
+the `[flux]` section.
+You shouldn't have to change the default settings but can if you prefer.
+
+_**influxdb.conf**_  
+```toml
+# ...
+
+[flux]
+  # Determines whether the RPC service is enabled.
+  enabled = true
+
+  # Determines whether additional logging is enabled.
+  log-enabled = true
+
+  # The bind address used by the Flux RPC service.
+  bind-address = ":8082"
+
+# ...
+```
+
+### Configure the storage service on InfluxDB
+In future versions of InfluxDB, the default `index-version` will be switching from the
+in-memory [Time-Structured Merge Tree (TSM)](https://docs.influxdata.com/influxdb/v1.5/concepts/storage_engine/)
+engine to the [Time Series Index (TSI)](https://docs.influxdata.com/influxdb/v1.5/concepts/tsi-details/) engine,
+which stores indexed data on disk.
+
+Update your the `index-version` setting under the `[data]` section in your
+`influxdb.conf` to use `tsi1`:
+
+```toml
+# ...
+[data]
+  # ...
+  index-version = "tsi1"
+  # ...
+```
+
+### Start InfluxDB, Flux, and Chronograf
+InfluxDB, Flux, and Chronograf are all run as separate daemonized processes and must be started separately.
+Run each of the following commands in their own terminal sessions.
+
+```bash
+# Start the influxd daemon
+influxd -config path/to/influxdb.conf
+```
+```bash
+# Start the fluxd daemon
+fluxd
+```
+```bash
+# Start Chronograf
+chronograf
+```
+
+### Configure Chronograf
+Open Chronograf in your browser of choice at [localhost:8888](http://localhost:8888].
+
+#### Connect to InfluxDB
+If not already connected to InfluxDB, you will be prompted for connection details.
+Provide the necessary credentials and save.
+
+![Connect Chronograf to InfluxDB](/images/connect-to-influxdb.png)
+
+#### Connect to Flux
+To connect Chronograf to your running `fluxd` process, click on the Flux Editor icon in the left navigation.
+The pre-populated credentials should work unless the `[flux]` settings were changed
+from their defaults or you've aliased Flux to a different domain.
+
+![Flux Editor Button](/images/flux-editor-btn.png)
+![Connect Chronograf to Flux](/images/connect-to-flux.png)
+
+Once the connection is established, the Flux Editor will display.
+
+## Using Flux with the Sandbox
+To use Flux with the [InfluxDB Sandbox](https://github.com/influxdata/sandbox),
+start the sandbox with the `-nightly` flag to pull the nightly builds of Influx services.
+
+```bash
+./sandbox up -nightly
```

Comment:
checked a few branches.  there's no `-nightly` flag implemented in sandbox

Reply:
I'm working on it! ^